lib/new_relic/language_support.rb

```diff
@@ -73,6 +73,11 @@ def constantize(const_name)
       end
     end
 
+    def camelize(string)
+      camelized = string.downcase
+      camelized.split(/\-|\_/).map(&:capitalize).join
```

Comment:
Hmm... I was just about to say that you don't need to use escaping here and just go with `/-|_/`, but according to `Regexp.escape`'s output, it does like to escaping the `-` character. TIL.